plugins/events/create.js

```diff
@@ -182,6 +182,7 @@ module.exports = () => ({
 
                                         payload.prInfo = prInfo;
                                         payload.prRef = prInfo.ref;
+                                        payload.chainPR = pipeline.prChain;
```

Comment:
probably too late to change this since it's already in the `models`, but not sure why we name they key `chainPR` instead of `prChain`

Reply:
I think it was to be consistent with `restrictPR`